specification/sql/resource-manager/readme.md

```diff
@@ -52,7 +52,7 @@ input-file:
 - Microsoft.Sql/stable/2014-04-01/serviceObjectives.json
 - Microsoft.Sql/stable/2014-04-01/sql.core_legacy.json
 - Microsoft.Sql/stable/2014-04-01/usages_legacy.json
-- ./Microsoft.Sql/preview/2020-11-01-preview/BackupShortTermRetentionPolicies.json
+- ./Microsoft.Sql/preview/2021-02-01-preview/BackupShortTermRetentionPolicies.json
```

Comment:
please put this 2021-02-01-preview line to Line 148 (the bottom section of v5 sdk tag) with alphabetical order.

Reply:
moved